app/controllers/api/v1/checkpoints_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Api::V1::CheckpointsController < ApplicationController
+
+  def show
+    checkpoint = Checkpoint.find_by(ethereum_address: params[:id])
```

Comment:
Is `params[:id]` based on the route wildcard? I hadn't noticed this mismatch before. @ski-climb